onnxscript/version_converter/_version_converter.py

```diff
@@ -220,6 +220,38 @@ class _VersionConverter:
     def __init__(self, target_version: int):
         self.target_version = target_version
 
+    def _maybe_set_opset_version(
+        self, model_or_function: ir.Model | ir.Function, domain: str, version: int | None
+    ):
+        """Set the opset version for the domain."""
+        current_version = model_or_function.opset_imports.get(domain)
+        if version is None or current_version is None:
+            return
+        if domain == "":
+            model_or_function.opset_imports[domain] = max(version, current_version)
+            return
+        elif domain == "ai.onnx":
+            model_or_function.opset_imports[domain] = max(version, current_version)
+            return
```

Comment:
```suggestion
        if domain == "":
            model_or_function.opset_imports[domain] = max(version, current_version)
            return
        elif domain == "ai.onnx":
            model_or_function.opset_imports[domain] = max(version, current_version)
            return
```

since `""` and `"ai.onnx"` is the same, they should share the same opset import. In practice it may be good to assert that we don't have any `"ai.onnx"` nodes if handling both domains over complicates the logic

Reply:
Hmmm ... I guess #2283 does NOT do it for opset-imports, that remains a TODO, is that right?

Reply:
Yeah we need to create a new data structure for opset imports to do this, which is a todo.